app/dashboard/migrations/0195_auto_20211122_2034.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 2.2.24 on 2021-11-22 20:34
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dashboard', '0194_auto_20211029_1748'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='searchhistory',
+            options={'verbose_name_plural': 'Search History'},
+        ),
+    ]
```

Comment:
This was generated when I did the make migrations command for the additional image field on grant_branding_routing_policy object. I'm not sure if this is needed or not, but left it in for now. Let me know if I should remove it.

Reply:
Let's remove it and see if a migration was missed in master